Jenkinsfile

```diff
@@ -50,8 +52,7 @@ void job_step_update(def value) {
 }
 
 // For master, this is just some wildly high number
-/* groovylint-disable-next-line CompileStatic */
-String next_version = '2.3'
+String next_version = '1000'
```

Comment:
I don't know what your intentions are for this PR but this above change **CANNOT** land.